src/datasets/search.py

```diff
@@ -757,8 +763,10 @@ def get_nearest_examples_batch(
                 The number of examples to retrieve per query.
 
         Returns:
-            - total_scores (`List[List[float]`): The retrieval scores of the retrieved examples per query.
-            - total_examples (`List[dict]`): The retrieved examples per query.
+            `(total_scores, total_examples)`:
+                A tuple of `(total_scores, total_examples)` where:
+                - **total_scores** (`List[List[float]`): the retrieval scores (`IndexFlatL2` score by default) of the retrieved examples per query
+                - **total_samples** (`List[dict]`): the retrieved examples per query
```

Comment:
Is this supposed to be `total_samples` (see [line 770](https://github.com/huggingface/datasets/blob/b8067c0262073891180869f700ebef5ac3dc5cce/src/datasets/search.py#L770)) instead of `total_examples`?

Reply:
ideally we should be consistent and always use "examples" instead of "samples"